src/main/java/net/sf/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -135,6 +135,15 @@ public static BibEntry singleFromString(String bibtexString,
         return entries.iterator().next();
```

Comment:
make singleFromString deprecated and rewrite it as `return singleFromStringOptional(...).orElse(null)`;
Then this can be merged.